proto/buf/alpha/registry/v1alpha1/search.proto

```diff
@@ -16,6 +16,16 @@ syntax = "proto3";
 
 package buf.alpha.registry.v1alpha1;
 
+enum SearchFilter {
+  SEARCH_FILTER_UNSPECIFIED = 0;
+  SEARCH_FILTER_REPOSITORY = 1;
+  SEARCH_FILTER_ORGANIZATION = 2;
+  SEARCH_FILTER_USER = 3;
+  SEARCH_FILTER_TEAM = 4;
+  SEARCH_FILTER_PLUGIN = 5;
+  SEARCH_FILTER_TEMPLATE = 6;
```

Comment:
Nittest of nits: the order of this is a bit curious to me, as we traditionally group things like so:

```
(user, organization)
(repository, plugin, template)
(team)
```

Reply:
i definitely didn't have a good reason to have it in the previous order so i just reordered to match better with the traditional grouping!